admin/class-admin-init.php

```diff
@@ -653,30 +653,4 @@ private function dismiss_notice( $notice_name ) {
 	private function has_postname_in_permalink() {
 		return ( false !== strpos( get_option( 'permalink_structure' ), '%postname%' ) );
 	}
-
-	/********************** DEPRECATED METHODS **********************/
```

Comment:
I'm not sure if we want to remove this header as we might have deprecated methods in the feature.

Reply:
I wondered the same but decided to remove the header because other functions without deprecated functions also don't have this header.